README.md

```diff
@@ -45,6 +44,12 @@ We the People is a work in progress and currently exists at a very basic level o
 
 The following descriptions are for informational purposes only and should not be interpreted as commitments or guarantees of future code releases in any way.
 
+*Move from MongoDB to MySQL*
```

Comment:
The inclusion of this, and the other mongo comments, make it sound like we're moving back to mongo rather than away from it.